content/networking/security-groups.md

```diff
@@ -68,3 +68,15 @@ column of assigned groups.
 After clicking save, your instance will be accessible on port 80 and 443. To
 confirm this, you can open your instance details and see the list of open ports
 will now include the assigned web ports!
+
+## ICMP / Ping Rules
+
+In order to allow servers to ping each other, or be reached via ICMP through
+the internet, you must create a Security Group Rule of protocol **ICMP**, type
+**-1**, and code **-1**.
+
+![Security Group Rule ICMP](/img/content/networking/security-group-icmp.png)
+
+The above screenshot shows an example of opening ICMP traffic to all internal
+traffic on the `10.0.0.0/24` subnet, which will only allow the traffic for
```

Comment:
agreed, it should probably be more generic. Although the 10.0.0.0/24 subnet is good to have as an example